CMakeLists.txt

```diff
@@ -44,11 +44,25 @@ else()
     set(SFML_BUILD_EXAMPLES FALSE)
 endif()
 
+# add options to select which modules to build
+sfml_set_option(SFML_BUILD_SYSTEM TRUE BOOL "TRUE to build SFML's System module. This setting is ignored, if any of the other modules is built.")
```

Comment:
What's the point of not building sfml-system? I think we can remove this option.